ocaml/src/remotes.ml

```diff
@@ -132,13 +132,13 @@ module Pool = struct
 
     type t = {
       pools : (Osd.id, osd_pool) Hashtbl.t;
-      get_osd_kind : Osd.id -> OsdInfo.kind Lwt.t;
+      get_osd_kind : Osd.id -> OsdInfo.kind Lwt.t; (* TODO: should not have _ Lwt.t signature  *)
```

Comment:
disagree with the comment you added here: we may have to get that info from the albamgr, so the `_ Lwt.t` signature here is expected imo.